internal/backend/test/tests.go

```diff
@@ -538,7 +543,19 @@ func (s *Suite) TestSave(t *testing.T) {
 
 	// wrap the tempfile in an errorCloser, so we can detect if the backend
 	// closes the reader
-	err = b.Save(context.TODO(), h, errorCloser{t: t, l: int64(length), ReadSeeker: tmpfile})
+	var beHash []byte
+	if b.Hasher() != nil {
+		beHasher := b.Hasher()
+		// must never fail according to interface
+		_, _ = beHasher.Write(data)
```

Comment:
I'd rather panic here in the unlikely case that `Write()` returns a non-nil error:
```go
_, err := beHasher.Write(data)
if err != nil {
    panic(err)
}
```

Reply:
I think it's never possible to receive an error here. But a sanity check won't hurt. -> I've added the check.